### DISCUSSION TITLE
Can't run on M1

### DISCUSSION THREAD

**Discussion by lexizli** [nigification/Flowgorithm-Mac-OSX#12] [Q&A]
Ventura 13.1
MacBook Apple M1 Pro

Try to use both install methods — I saw app, try to start, saw application in system monitor, but it hasn't any windows :-(

**Comment by jostasik** [comment]
Sorry for the issues! This was previously fixed with this update: https://github.com/jostasik/Flowgorithm-macOS/releases/tag/v3.4.2
